view/index.css

```diff
@@ -1,32 +1,22 @@
-table {
-    border-collapse: collapse;
-}
-
-table,
-th,
-td {
-    border: 1px solid black;
+.box {
+background-color: #009988;
+    margin: 15px;
 }
 
-.divTable {
-    display: table;
-    width: auto;
-    background-color: #eee;
-    border: 1px solid #666666;
-    border-spacing: 5px;
-    /*cellspacing:poor IE support for  this*/
-    /* border-collapse:separate;*/
-}
-
-.divRow {
-    display: table-row;
-    width: auto;
+.row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 100%;
+    border: 1px solid #000;
+    border-collapse: collapse;
 }
 
-.divCell {
-    float: left;
-    /*fix for  buggy browsers*/
-    display: table-column;
-    width: 200px;
-    background-color: #ccc;
+.column {
+    display: flex;
+    flex-direction: column;
```

Comment:
Basically you can rename this class to `.cell`, because it does not contain anything except `cell` data, and in this case `flex-direction: column` serves no purpose either and can be removed.